orgs.js

```diff
@@ -37,6 +37,10 @@
           "type": "org"},
         {"name": "Watson-Explorer",
           "type": "org"},
+	{"name": "timlynar/Containerized-Forecasting-Workflow",
```

Comment:
you need to submit either a) a single repo, or b) an org. In this case I think you want the repo. Suggest you remove lines 40-41